The build instructions do not mention clr.dactools (soon nativeprereqs)

Hi I am hitting on this issue with latest version:

The application to execute does not exist: 

**9>The application to execute does not exist: 'E:/work/dotnet/runtime/artifacts/bin/coreclr/windows.x64.Debug/DacTableGen/DacTableGen.dll'
9>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for 'E:\work\dotnet\runtime\artifacts\obj\coreclr\windows.x64.Debug\ide\CMakeFiles\1f2a7b96d174e6afb14c443eb282e277\inject_debug_resources_singlefilehost.timestamp.rule;E:\work\dotnet\runtime\artifacts\obj\coreclr\windows.x64.Debug\ide\CMakeFiles\1b6f9782b401a901cd7a6d111bc248c5\inject_debug_resources_singlefilehost.rule' exited with code -2147450751.**

This seems to be related to DAC/ninja.
Any idea what is the real issue here?

--- a/docs/workflow/debugging/coreclr/debugging-runtime.md
+++ b/docs/workflow/debugging/coreclr/debugging-runtime.md
@@ -41,6 +41,7 @@ If for some reason `System.Private.CoreLib.dll` is missing, you can rebuild it w
 
 Visual Studio's capabilities as a full IDE provide a lot of help making the runtime debugging more amiable.
 
+0. Run `.\build.cmd clr.nativeprereqs -a <architecture> -c <configuration>`. This will build some of the tools requiremented for the native build. This step only needs to be run once as long you don't clean the `artifacts` directory.
 1. Open the CoreCLR solution _(coreclr.sln)_ in Visual Studio.
    * _Method 1_: Use the build scripts to open the solution:
       1. Run `.\build.cmd -vs coreclr.sln -a <architecture> -c <configuration>`. This will create and launch the CoreCLR solution in VS for the specified architecture and configuration. By default, this will be `x64 Debug`.

